flit_core/README.rst

```diff
@@ -1,6 +1,9 @@
 flit_core
 ---------
 
-This provides a PEP 517 build backend for packages using Flit.
-The only public interface is the API specified by PEP 517, at ``flit_core.buildapi``.
+This provides a `PEP 517 <https://peps.python.org/pep-0517/>`_ build backend
```

Comment:
It now also covers PEP 660, which is an extension of the spec. @pradyunsg I don't think we yet have a generic reference page for the build system API specified by these two PEPs?

Reply:
That's a WIP: https://github.com/pypa/packaging.python.org/pull/1111

Reply:
OK, then I'll leave this for now and we can remember to change it when that page is ready.

Reply:
Thanks. I'll leave this as is for now then, and we can update it when that page is ready.